test/integration/backward_compatible/parallel/sql/DataTypeTest.js

```diff
@@ -814,4 +815,62 @@ describe('Data type test', function () {
             rows[i]['__key'].should.be.eq(expectedKeys[i]);
         }
     });
+    it('should be able to decode/serialize JSON', async function () {
+        const inputs = [new HazelcastJsonValue(JSON.stringify({age: 3})), {age: 3}];
+        for (const input of inputs) {
+            await basicSetup(this);
+            const SqlColumnType = TestUtil.getSqlColumnType();
+
+            // JSON support is added in 5.1.
+            TestUtil.markClientVersionAtLeast(this, '5.1');
+            TestUtil.markServerVersionAtLeast(this, client, '5.1');
```

Comment:
By top do you mean before or beforeEach? This JSON datatype needs additional skip logic since it is added later than others added. If you mean changing these two lines' orders, no problem I did it.

Reply:
I meant line 819, just after the `it` block, so no code is executed if the checks do not pass

Reply:
We cannot run it before `basicTests()` since the client would be uninitialized. Putting it right below it.